javascript-prototypes/warner-bros.js

```diff
@@ -0,0 +1,43 @@
+/* exported yakko, wakko, dot */
+
+var yakko = {
+  age: 14,
+  name: 'Yakko',
+  role: 'brother',
+  // describe: function () {
+  //   var description = this.name + ' is a ' + this.age + '-year-old Warner ' + this.role + '.';
+  //   return description;
+  // }
+};
+
+var wakko = {
+  age: 11,
+  name: 'Wakko',
+  role: 'brother',
+  // describe: function () {
+  //   var description = this.name + ' is a ' + this.age + '-year-old Warner ' + this.role + '.';
+  //   return description;
+  // }
+};
+
+var dot = {
+  age: 10,
+  name: 'Dot',
+  role: 'sister',
+  // describe: function () {
+  //   var description = this.name + ' is a ' + this.age + '-year-old Warner ' + this.role + '.';
+  //   return description;
+  // }
+};
+
+var description = {
+  describe: function () {
+    var description = this.name + ' is a ' +
+      this.age + '-year-old Warner ' + this.role + '.';
+    return description;
+  }
+}
```

Comment:
Please pay attention to what the instructions ask you to name things in the future:

---

![Screenshot_2](https://user-images.githubusercontent.com/16143771/111412285-60f94800-8699-11eb-8d77-20816eb187cb.jpg)